apps/web/public/static/locales/en/common.json

```diff
@@ -2838,6 +2838,12 @@
   "filter_operator_ends_with": "Ends with",
   "filter_operator_is_empty": "Is empty",
   "filter_operator_not_empty": "Not empty",
+  "filter_operator_eq": "=",
+  "filter_operator_neq": "≠",
+  "filter_operator_gt": ">",
+  "filter_operator_gte": "≥",
+  "filter_operator_lt": "<",
+  "filter_operator_lte": "≤",
```

Comment:
NIT: IM not sure these need to be translated. I don't think anywhere in the world uses different symbols to this

Reply:
ah that's a good point!